dll/win32/comdlg32/filedlg.c

```diff
@@ -3096,9 +3096,10 @@ BOOL FILEDLG95_OnOpen(HWND hwnd)
 	else
 	  fodInfos->ofnInfos->Flags &= ~OFN_READONLY;
 
-        /* Attach the file extension with file name*/
+        /* if the user don't supply a extension, attach default extension to file name*/
```

Comment:
This is Wine code. If you want to modify only comment here, it's better to do inside a `__REACTOS__` guard below.

Reply:
yes